includes/definitions/discovery/opengear.yaml

```diff
@@ -0,0 +1,65 @@
+mib: OG-STATUSv2-MIB
+modules:
+    sensors:
+        pre-cache:
+            data:
+                - oid:
+                    - ogCellModemTable
+                    - ogEmdTemperatureTable
+        temperature:
+            data:
+                -
+                    oid: ogEmdTemperatureTable
+                    value: ogEmdTemperatureValue
+                    num_oid: '.1.3.6.1.4.1.25049.17.9.1.5.{{ $index }}'
+                    descr: ogEmdTemperatureName
+                    index: 'ogEmdTemperatureValue.{{ $index }}'
+        signal:
+            data:
+                -
+                    oid: ogCellModemTable
+                    value: ogCellModem3gRssi
+                    num_oid: '.1.3.6.1.4.1.25049.17.17.1.11.{{ $index }}'
+                    descr: 'Cellular 3G Modem RSSI'
+                    index: 'ogCellModem3gRssi.{{ $index }}'
```

Comment:
To avoid people losing data, `index` should be `11.{{ $index }}`

Reply:
Done